selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/core/CheSeleniumMultiUserModule.java

```diff
@@ -39,6 +41,7 @@
   protected void configure() {
     bind(TestAuthServiceClient.class).to(KeycloakTestAuthServiceClient.class);
     bind(TestMachineServiceClient.class).to(CheTestMachineServiceClient.class);
+    bind(TestWorkspaceServiceClient.class).to(CheTestWorkspaceServiceClient.class);
```

Comment:
We can omit binding `TestWorkspaceServiceClient` here so as we have already done it in `CheSeleniumSuiteModule.java`.